examples/merkle_inclusion.masm

```diff
@@ -0,0 +1,17 @@
+# In this example we verify 1000 merkle paths in a sparse Merkle tree.
+# Every leaf on position i looks like [i, 0, 0, 0].
+begin
+    repeat.1000
+        # stack = [d, i, R, ...]
+        # duplicate the index and use as counter
+        dup.1 padw drop swap.3 
+        # => [i, 0, 0, 0, d, i, R,...]
+
+        # verify merkle path for node i
+        mtree_verify
+        # => [i, 0, 0, 0, d, i, R ...]
+
+        push.1 add swap.5 dropw
+        # => [d, i+1, R, ...]
+    end
```

Comment:
Is there a reason to do both `padw` and then `dropw` inside the loop? I think the following could work just as well:
```
begin
    repeat.1000
        # stack = [i, 0, 0, 0, d, i, R, ...]

        # verify merkle path for node i
        mtree_verify
        # => [i, 0, 0, 0, d, i, R ...]

        add.1 swap.5 add.1
        # => [i+1, 0, 0, 0, d, i+1, R, ...]
    end
end
```